core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -557,6 +559,7 @@ object Boot extends IOApp with LazyLogging {
                                            workspaceManagerDAO,
                                            workspaceManagerResourceMonitorRecordDao
           ),
+          billingProjectDeletion,
```

Comment:
Billing project and billing profile deletion methods are moved out of `BillingProjectLifecycle` and into `BillingProjectDeletion`.

`BillingProjectOrchestrator` uses `billingProfileManagerDAO` to get hints about cloud platform, though arguably we could remove that.